src/features/import-projects/garden/helpers.ts

```
@@ -0,0 +1,68 @@
+const convertIpfsHashToHttps = (hash: string) => {
+  return `${IPFS_GATEWAY}/${hash}`;
+};
+
+const getDescription = async (covenantIpfsHash: string) => {
+  if (!covenantIpfsHash) return "";
+  try {
+    const response = await fetch(`${IPFS_GATEWAY}/${covenantIpfsHash}`);
+
+    const contentType = response.headers.get("content-type");
+
+    if (contentType && contentType.includes("application/json")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)